.circleci/config.yml

```diff
@@ -420,49 +435,95 @@ jobs:
           keys:
             - '{{ .Environment.CIRCLE_CACHE_VERSION }}-bundle-2.4-{{ checksum ".circleci/bundle_checksum" }}'
       - *step_run_all_tests
+  deploy-release:
+    <<: *job_defaults
+    docker:
+      - *container-2_4
+    steps:
+      - checkout
+      - run:
+          command: |
+            apt-get -y -qq update
+            apt-get -y -qq install awscli
+      - restore_cache:
+          key: 'v1-bundle-2.4-{{ checksum "Gemfile" }}'
```

Comment:
Doing a checksum on `Gemfile` is actually insufficient: the `Gemfile` does not define the dependencies for the gem. If we updated the dependencies in the gemspec, it would attempt to restore an old cache missing gems.

`v1` also needs to be replaced with `{{ .Environment.CIRCLE_CACHE_VERSION }}`, because if we need to invalidate/clear the cache, CircleCI gives us no means to do so. We have to change this variable to force the cache to invalidate.